### ISSUE TITLE
[innerHTML] causes the server to duplicate content

### ISSUE THREAD

**Issue by kevinphelps** [angular/angular#19278]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior
Binding to `[innerHTML]` causes the server to render the html content in both the `innerHTML` property and in the element body.

```html
<div [innerHTML]="'content'"></div>
```

produces

```html
<div innerHTML="content">content</div>
```

## Expected behavior
The `innerHTML` property in the rendered HTML is not needed as far as I can tell. I added code to my project to remove it after Angular renders the page. Everything seems to work fine, and the HTML payload is much smaller.

## Minimal reproduction of the problem with instructions
See above.

## What is the motivation / use case for changing the behavior?
The rendered HTML payload is bloated with the duplicate content.

## Environment

<pre><code>
Angular version: 5.0.0-beta.5

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 8.5.0
- Platform:  Windows

Others:

</code></pre>

**Comment by Toxicable** [comment]
Confirmed on v4.4.2 - Which still uses parse5
This template:
```
  <div [innerHTML]="content"></div>

  content = '<span>some content</span>'
```
renders to 
```
    <div innerHTML="<span>some content</span>"><span>some content</span></div>
```

This may be changed by the change to Domino but I am unable to verify that since I cannot build on the latest beta version.

**Comment by kevinphelps** [comment]
If someone points me to the right place, I can work on PR to fix this.

**Comment by kevinphelps** [comment]
This is my workaround if anyone is curious or needs it: https://gist.github.com/kevinphelps/67caf1a280ceb3109a51b5ee332b4516

**Comment by kevinphelps** [comment]
This is my workaround if anyone is curious or needs it: https://gist.github.com/kevinphelps/20ea29ba8458d75befa195005a72310f

**Comment by masoudline** [comment]
my angular version is 5.2.9 and these issues exist now!
how can I remove innerHtml attribute manually ??

**Comment by kevinphelps** [comment]
This was fixed in Angular 6.1.0. If you cannot update Angular, see my workaround above.
